plugins/aver-hid/fu-aver-hid-device.c

```diff
@@ -80,6 +81,18 @@ fu_aver_hid_device_poll(FuDevice *device, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_aver_hid_device_check_dual_isp(FuAverHidDevice *device)
```

Comment:
I think a much better way to do this is to use:

    # AVer VB342Pro
    [USB\VID_2574&PID_0AB1]
    Plugin = aver_hid
    Flags = dual-isp

and then use `fu_device_register_private_flag` in the plugin, and then check something like `fu_device_has_private_flag(device, FU_AVER_HID_FLAG_DUAL_ISP)` to check that flag.